### ISSUE TITLE
Namespaced aliases lost in translation

### ISSUE THREAD

**Issue by Schahen** [Kotlin/dukat#119]
While converting
```typescript
declare var _: _.StaticInstance;
export = _;

declare module _ {
    type PropertyShorthand = string | number;

    interface StaticInstance {
        fetch<T>(param: _.PropertyShorthand): T[];
    }
}
```
StaticInstance converts to:
```kotlin
external interface StaticInstance {
    fun <T> fetch(param: `_`.PropertyShorthand): Array<T>
}
```

While PropertyShorthand is (ha!) lost in translation.

**Comment by Schahen** [comment]
An other example, quite shameful one as well
```
declare module a {
  function ping(a: b.AlphaNumeric);
}

declare module b {
  type AlphaNumeric = string | number
}

export = a.ping
```

**Comment by Schahen** [comment]
resolved in 0.0.19
